process_jartic_simple_v2.py

```
@@ -0,0 +1,205 @@
+    print(f"Date range: {args.start} to {args.end}")
+    
+    # Count total measurements unless skipped
+    if args.skip_count and args.estimated_total:
+        total_to_process = args.estimated_total
+        print(f"Using estimated total: {total_to_process:,}")
+    else:
+        total_to_process = count_total_measurements(filtered_archives, start_date, end_date)
+    
+    # Prepare output
+    storage = DataStorage()
+    timestamp = datetime.now().strftime('%Y%m%d_%H%M%S')
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)